audio/nrsc5/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        theori-io nrsc5 f1a32a551471dc5045193027e4c828746c5790fd
+
+revision            0
+
+categories          audio
+maintainers         {netjibbing.com:blake @trodemaster}
+license             GPL
+
+description         This program receives NRSC-5 digital radio stations using an RTL-SDR dongle
+long_description    \
+This program receives NRSC-5 digital radio stations using an RTL-SDR dongle. It offers a command-line interface as well as an API upon which other applications can be built.
+
+checksums           rmd160  03b316b7c1bbc38966758958c2d4f35aec32e235 \
+                    sha256  7704739e40dc9467dbdf1f484ca87414b3c54f8f63c8eb1f1920729d3addb5cc \
+                    size    23168025
+
+patchfiles         CMakeLists.txt.diff
+
+configure.args-append \
+    -DUSE_NEON=ON \
+    -DUSE_SSE=ON \
+    -DUSE_FAAD2=ON
+
+depends_build       port:libtool \
```

Comment:
This would be best as `depends_build-append`.  Especially when you're using portgroups, implicit dependencies may be added by portgroups, and using the `-append` suffix is safer for all `depends_*` specifications.